### PR TITLE
Update eddsa to 0.2.0 and sshd-core to 1.6.0

### PR DESCRIPTION
**Pull request by paladox**

### CODE REVIEW

**Comment by charlesrgould** [comment]
Thanks but I'm curious, why one commit per file?

**Comment by paladox** [comment]
@charlesrgould i am using the inline edit feature. It seems it wont allow you to add files and create one commit through the web ui.

**Comment by paladox** [comment]
Hmm this seems to require java 8. It seems that you still support java 6+.

**Comment by paladox** [comment]
im nto sure why it failed java 9. but it passes for 8.

File: build.gradle [hunk 1]
```diff
@@ -23,8 +23,8 @@ repositories {
   }
 }
 
-sourceCompatibility = 1.6
-targetCompatibility = 1.6
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
```
Comment (charlesrgould) [inline build.gradle]: Not my decision, but I highly doubt the project owner will merge if indeed the change requires Java 8.

**Comment by hierynomus** [comment]
Indeed, I will not merge this exactly due to this. There are users still depending on Java1.6 support.

**Pull request closed without merging**